### DISCUSSION TITLE
严阵以待最新版本清单+密钥（12.20更新）

### DISCUSSION THREAD

**Discussion by 1138568406** [pjy612/SteamManifestCache#113] [Share]
appid:1144200 keys:1144201;b9236894ba071d12dd117ce16220fefeb20574d7c6f7d2170e3113346867f3d6
[严阵以待.zip](https://github.com/pjy612/SteamManifestCache/files/13749088/default.zip)
内测码：28BB26F319CED744
邮箱：1138568406@qq.com

**Comment by pjy612** [comment]
这个被分享过了吧

**Reply by 1138568406** [reply]
20号游戏更新之后那个就不知道为啥不能用了，又重新租了个号整的清单

**Reply by pjy612** [reply]
更新肯定不会那么及时的，但是 对照下steamdb 也不会太慢

**Reply by 1138568406** [reply]
这个版本就是最新可以用的了

**Reply by pjy612** [reply]
说的是被人分享过密钥
